src/quest/extras/dynamodb.py

```diff
@@ -0,0 +1,119 @@
+import os
+from .. import BlobStorage, Blob, WorkflowManager, WorkflowFactory, PersistentHistory, History
```

Comment:
I don't love these dependencies. Should we put the `create_*_manager()` functions somewhere else so all the extra files aren't depending on the same things over again?